e3sm_diags/driver/annual_cycle_zonal_mean_driver.py

```diff
@@ -143,40 +133,45 @@ def _run_diags_annual_cycle(
             parameter.regrid_method,
         )
 
-        test_zonal_mean = spatial_avg(ds_test, var_key, axis=["X"], as_list=False)
-        test_reg_zonal_mean = spatial_avg(
-            ds_test_reg, var_key, axis=["X"], as_list=False
+        test_zonal_mean: xr.DataArray = spatial_avg(ds_test, var_key, axis=["X"], as_list=False)  # type: ignore
+        test_reg_zonal_mean: xr.DataArray = spatial_avg(
+            ds_test_reg, var_key, axis=["X"], as_list=False  # type: ignore
         )
 
         if (
             parameter.ref_name == "OMI-MLS"
         ):  # SCO from OMI-MLS only available as (time, lat)
             test_zonal_mean = test_zonal_mean.sel(lat=(-60, 60))
             test_reg_zonal_mean = test_reg_zonal_mean.sel(lat=(-60, 60))
+
             if var_key == "SCO":
                 ref_zonal_mean = ds_ref[var_key]
                 ref_reg_zonal_mean = ds_ref[var_key]
             else:
-                ref_zonal_mean = spatial_avg(ds_ref, var_key, axis=["X"], as_list=False)
+                ref_zonal_mean = spatial_avg(ds_ref, var_key, axis=["X"], as_list=False)  # type: ignore
                 ref_reg_zonal_mean = spatial_avg(
-                    ds_ref_reg, var_key, axis=["X"], as_list=False
+                    ds_ref_reg, var_key, axis=["X"], as_list=False  # type: ignore
                 )
 
         else:
-            ref_zonal_mean = spatial_avg(ds_ref, var_key, axis=["X"], as_list=False)
+            ref_zonal_mean = spatial_avg(ds_ref, var_key, axis=["X"], as_list=False)  # type: ignore
             ref_reg_zonal_mean = spatial_avg(
-                ds_ref_reg, var_key, axis=["X"], as_list=False
+                ds_ref_reg, var_key, axis=["X"], as_list=False  # type: ignore
             )
+
         # Make a copy of dataset to preserve time dimension
         diff = test_reg_zonal_mean.to_dataset().copy()
         diff[var_key].values = test_reg_zonal_mean.values - ref_reg_zonal_mean.values
 
+        parameter._set_param_output_attrs(
+            var_key, "ANNUALCYCLE", region, ref_name, ilev=None
+        )
         _save_data_metrics_and_plots(
             parameter,
             plot_func,
             var_key,
             test_zonal_mean.to_dataset(),
             ref_zonal_mean.to_dataset(),
             diff,
-            metrics_dict={},
+            metrics_dict=None,
```

Comment:
`metrics_dict` can be set to `None` after removing the `metrics_dict` arg from the plot function.